packages/ui/src/components/Table/BaseTable.tsx

```diff
@@ -158,25 +158,27 @@ const BaseTable = <T extends GenericRowStructure>(props: BaseTableProps<T>) => {
 
     return (
         <div className={classes.root}>
-            <FiltersRowContainer className={classes.headArea}>
-                {makeSearchableRowContent || searchProps ? (
-                    <div className={classes.searchInputContainer}>
-                        {renderSearchInput ? (
-                            renderSearchInput({
-                                searchValue: searchInputProps.value,
-                                handleChangeSearchValue: searchInputProps.onChange,
-                            })
-                        ) : (
-                            <TextField
-                                placeholder={searchInputPlaceholder}
-                                onChange={(e) => searchInputProps.onChange(e.target.value)}
-                                value={searchInputProps.value}
-                            />
-                        )}
-                    </div>
-                ) : null}
-                <div className={classes.actionsContainer}>{renderTableActions?.(selectedRowIds)}</div>
-            </FiltersRowContainer>
+            {makeSearchableRowContent || searchProps || renderTableActions ? (
+                <FiltersRowContainer className={classes.headArea}>
+                    {makeSearchableRowContent || searchProps ? (
```

Comment:
how are we handling here the case where we default the searchInputProps to something ? Is that default object ever going to be rendered ?

Reply:
Well... that's for who wrote this to figure it out why only checked the searchProps. But yeah, the default would never be rendered based on this